src/lib/logSummary/logSummary.cpp

```diff
@@ -122,6 +122,13 @@ static void* logSummary(void* vP)
     if (biRaisedNew   < 0)  { biRaisedNew   = LONG_MAX - biRaisedInLastSummary   + biRaised;   }
     if (biReleasedNew < 0)  { biReleasedNew = LONG_MAX - biReleasedInLastSummary + biReleased; }
 
+    //
+    // No log summary if everything is ZERO
+    //
+    if (deRaised + deReleased + neRaised + neReleased + biRaised + biReleased == 0)
```

Comment:
Not sure if this a good idea... Event if there is no changes in the counter, user may want to have the trace be printed as a "keepaliving" feature.

Why this is needed (in the context of valgrind test suite fixing)?

Reply:
Same as always, due to slower execution under valgrind a 0-0-0-0-0-0 message is present under valgrind and not under normal functest.
Only way to fix it is to do what I did.
If a user needs this, simply "no metrics trace" means no metrics to report.
I would prefer to not make this change but between picking between
1. This change
and
2. Exclude this test from the valgrind test suite

I prefer option 1.

Reply:
I understand this only affect to one test (cases/1585_log_summary/log_summary.test). In that case, I'd suggest to use the same solution than in cases/1697_more_than_one_bad_input/more_than_one_bad_input.test, adding

```
# NOT VALGRIND READY ...
# The log summary is heavily dependent on the timing, which is totally off under valgrind ...
```

That is, option 2. As I mention, not sure of the impact of this change and it is preferable to be conservative.

Reply:
ok, seems like a bad idea in my opinion, but, I don't really mind.